util/lksconfig

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Allows to read/write the configuration of Andor Shamrock spectrograph
+'''
+Created on 07 October 2022
+
+@author: Canberk Akin
+
+Copyright © 2022 Canberk Akin, Delmic
+
+This file is part of Odemis.
+
+lksconfig is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+lksconfig is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+lksconfig. If not, see http://www.gnu.org/licenses/.
+'''
+# To test:
+# ./util/lksconfig --log-level 2 --channel 4 --set-pid 1 [2 [3]] --address 192.168.2.12  (/dev/ttyACM2)
+# ./util/lksconfig --channel 1 --get-pid --role sample-thermostat
+# ./util/lksconfig --log-level 2 --channel 4 --set-pid 1 2 3 --address /dev/fake
```

Comment:
this (last) line is for testing. 
The first two lines are exame lines. Please adjust the text to reflect that.
Also, drop the square brackets as they imply the values are optional, while finally you didn't implement support for that.